src/components/Layout/Page.tsx

```diff
@@ -153,7 +153,7 @@ export function Page({children, toc, routeTree, meta, section}: PageProps) {
                     <>
                       <div className="flex flex-col items-center m-4 p-4">
                         <p className="font-bold text-primary dark:text-primary-dark text-lg mb-4">
-                          How do you like these docs?
+                          Bagaimana Anda menyukai dokumentasi ini?
```

Comment:
```suggestion
                          Seberapa suka Anda dengan dokumentasi ini?
```

Shall we paraphrase it to feel more natural? 😁

Reply:
@zainfathoni Yep, let's do it.